src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+  title: { { title } }
+tags: quicktip
+---
+
+Even with the best of intentions, you may be making decisions that exclude disabled users. Below are practical, intentional steps you can start today to create and share more accessible social media, digital, and multimedia content.
+
+## Social Media Posts
+
+### Use capital letters in hashtags
+Punctuation helps text-to-speech tools like screen readers speak more accurately, and helps with readability overall. This includes in hashtags, where capital letters help distinguish words.
+
+> Make it a habit to #UseCapitalLetters rather than #hardtoreadlowercase
+
+### Don’t share just images of text
+Images of text (such as your favorite quote or saying) can be difficult for people with visual impairments to read and don’t allow people with reading difficulties to leverage text-to-speech read-aloud features. When posting these kinds of images, also write out the text in your post.
+
+<figure role="figure" aria-label="Typing out the text in an image makes the content more accessible.">
+	<img alt="Instagram post where the photo is a quote reading 'Whatever is good for your soul do that - Anurup' and the quote's text is also typed in the post." src="/img/posts/2021-06-14-Everyday_Accessibility/instagram_quote.jpg" />
+	<figcaption>Typing out the text in an image makes the content more accessible.</figcaption>
+</figure>
+
+### Use “alt text” features to describe images
+Pictures are significant in social media, but also inaccessible to visually impaired readers unless they have a text-based description. When uploading, find the “image description” or “alternative/alt text” feature in the software to write a description. For sample instructions, review [Alt Text for Social Media](https://symphonyagency.com/alt-text-for-social-media/). 
+
+When adding descriptions, don’t write every detail, just enough to ensure everyone can understand your post and share in the image’s meaning. For examples, read Scott Vinkle’s [Considerations when writing alt text](https://ux.shopify.com/considerations-when-writing-alt-text-a9c1985a8204).
+
+<figure role="figure" aria-label="Twitter's Add Description feature allows adding alt text for visually impaired readers.">
+	<img alt="Two mobile phone screenshots. The first shows a draft Twitter post reading, 'Coolest clouds I've ever seen' with a photo attached. The second shows the Image Description input screen with text reading 'A field of wildflowers with hills in the background and clouds overhead'." src="/img/posts/2021-06-14-Everyday_Accessibility/twitter_alt.png" />
+	<figcaption>Twitter's "Add description" feature allows adding "alt text" for visually impaired readers.</figcaption>
+</figure>
+
+### Limit your emojis
+Emojis are great for emphasis and playfulness, but too many and in unconventional placements can be distracting and hard to read. Keep your emojis to the minimum needed to make your point, group them together, and move them towards the end of any text.
```

Comment:
```suggestion
### Limit your emojis

Emojis are great for emphasis and playfulness, but too many and in unconventional placements can be distracting and hard to read. Keep your emojis to the minimum needed to make your point, group them together, and move them towards the end of any text.
```

Reply:
>distracting and hard to read

Do you think it's worth mentioning the announcement clutter created in screen reader announcements when emojis are over-applied?

Reply:
Sure, since it's related to the example as well (assuming I keep that example). I'll rephrase that first sentence to say something like "it's cluttered with text-to-speech and hard to read visually". That should address both use cases.